### PR TITLE
[BAHIR-245] Add git .asf.yaml file

### PR DESCRIPTION
**Pull request by eskabetxe**
more info in [here](https://issues.apache.org/jira/browse/BAHIR-245)

@lresende could you check

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -257,6 +257,7 @@
           </executions>
           <configuration>
             <excludes>
+              <exclude>.asf.yaml</exclude>
               <exclude>.gitignore</exclude>
```
Comment (lresende) [inline pom.xml:261]: You shouldn't exclude this.

File: .asf.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+github:
```
Comment (lresende) [inline .asf.yaml:1]: Yaml files allow asf license header using `#` at the beginning of the line.

Comment (eskabetxe) [reply .asf.yaml:1]: isnt this like '.gitignore' a configuration file...
this file wont be on released jars.

Comment (lresende) [reply .asf.yaml:1]: The configuration below on the pom disables license checking on files that do not allow license headers. all files that allow should have.

File: pom.xml (hunk 1 above)
Comment (eskabetxe) [reply pom.xml]: done

File: .asf.yaml (hunk 2 above)
Comment (eskabetxe) [reply .asf.yaml:18]: done

**Review by lresende — APPROVED** [review]

**Pull request merged**